net/acme/Makefile

```diff
@@ -72,18 +72,27 @@ define Package/acme-dnsapi/install
 	$(INSTALL_DATA) $(PKG_BUILD_DIR)/dnsapi/*.sh $(1)/usr/lib/acme/dnsapi
 endef
 
-define Package/luci-app-acme
+define Package/luci-app-acme/default
   SECTION:=luci
   CATEGORY:=LuCI
   TITLE:=ACME package - LuCI interface
-  DEPENDS:= lua luci-base +acme luci-app-uhttpd
+  DEPENDS:= lua luci-base +acme 
   SUBMENU:=3. Applications
 endef
 
+define Package/luci-app-acme
+  $(Package/luci-app-acme/default)
+  DEPENDS += luci-app-uhttpd
+endef
+
+Package/luci-app-acme-nginx = $(Package/luci-app-acme/default)
```

Comment:
Rather than splitting the package, just drop the dependency on luci-app-uhttpd from the main package

Reply:
think some type of alert should be placed in the luci-app page if uhttpd is installed(to alert that luci-app-uhttpd is available...)